src/components/forms/Range/Range.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+
+import Range from './Range';
+import { RangeProps } from './Range.types';
+
+export default {
+  title: 'components/forms/Range',
+  component: Range,
+  argTypes: {
+    min: { control: 'number' },
+    max: { control: 'number' },
+    defaultValue: { control: 'number' },
+    isProgressRight: { control: 'boolean' },
+    isDisabled: { control: 'boolean' },
```

Comment:
Please remove, Storybook is able to pick this from prop-types and TS annotations